source/_docs/automation.markdown

```diff
@@ -52,6 +52,7 @@ Actions are all about calling services. To explore the available services open t
 If you always want your automations to be enabled or disabled upon Home Assistant restart, then you have to set an initial state in your automations. Otherwise the previous state will be restored.
 
 If an automation is disabled (turned off) then it will never trigger. Only automations that are enabled (turned on) will trigger.
+To have your automation enabled you should either add `initial_state: true` to your automation description (it will always make it enabled on Home Assistant startup) or turn it on manually via UI/another automation/developer tools (that way the last stored state of the automation will be restored on the next Home Assistant startup).
```

Comment:
Isn't this covered already above (line 52)?

Reply:
I agree with @rohankapoorcom  on this @akasma74, it is already stated at line 52.

It would make more sense to extend the description of line 52 instead of adding it at this place (and kinda repeating things).

Reply:
Please have a look at the amended version.
The main reason of this PR is that the current text covers state's restoration , but says nothing about initial state of newly created automation without `initial_state` (that behaviour has changed recently to opposite), these 2 things are completely separate and should not be mixed up/omitted.
It also says nothing about HA startup failure.
Hope that makes more sense.